config.go

```diff
@@ -5,7 +5,14 @@ type Config struct {
 	ListenAddress                 string `yaml:"listen_address"`
 	SpanExpirationDurationSeconds string `yaml:"span_expiration_duration_seconds"`
 	StatsdAddress                 string `yaml:"statsd_address"`
-	WorkerCount                   int    `yaml:"worker_count"`
-	WorkerSpanDepth               int    `yaml:"worker_span_depth"`
-	WorkerWatchDepth              int    `yaml:"worker_watch_depth"`
+	TLS                           struct {
+		Certificates []struct {
+			Cert string `yaml:"cert"`
+			Key  string `yaml:"key"`
+		} `yaml:"certificates"`
+		ClientCas []string `yaml:"client_cas"`
```

Comment:
nit: `ClientCAs` (does the generator even let you do that?)

Reply:
lemme check, but i don't think so